x-pack/plugins/observability_solution/slo/common/locators/paths.ts

```diff
@@ -25,10 +26,13 @@ export const paths = {
   sloEdit: (sloId: string) => `${SLOS_BASE_PATH}/edit/${encodeURIComponent(sloId)}`,
   sloEditWithEncodedForm: (sloId: string, encodedParams: string) =>
     `${SLOS_BASE_PATH}/edit/${encodeURIComponent(sloId)}?_a=${encodedParams}`,
-  sloDetails: (sloId: string, instanceId?: string, remoteName?: string) => {
+  sloDetails: (sloId: string, instanceId?: string, remoteName?: string, tabId?: string) => {
     const qs = new URLSearchParams();
-    if (!!instanceId) qs.append('instanceId', instanceId);
+    if (!!instanceId && instanceId !== '*') qs.append('instanceId', instanceId);
```

Comment:
You should import ALL_VALUE from slo-schema.

`import { ALL_VALUE } from '@kbn/slo-schema';`